htdocs/admin/pdf.php

```diff
@@ -284,6 +285,10 @@
 print $form->selectyesno('MAIN_GENERATE_DOCUMENTS_WITHOUT_VAT', (!empty($conf->global->MAIN_GENERATE_DOCUMENTS_WITHOUT_VAT)) ? $conf->global->MAIN_GENERATE_DOCUMENTS_WITHOUT_VAT : 0, 1);
 print '</td></tr>';
 
+print '<tr class="oddeven"><td>'.$langs->trans("SowVATIsNotUsedForDocumentWithZeroVat").'</td><td>';
```

Comment:
I suggest to add a test here
if country_code == 'FR' && ! empty($mysoc->tva_assuj)

if no FR, mention will never appear.
If $mysoc->tva_assuj  is set, mention will already appear.

Reply:
Done.